test/java/scenarios.json

```diff
@@ -26,26 +26,30 @@
             },
             "java": {
                 "version": "latest",
+                "installAnt": true,
```

Comment:
Can we add tests to validate this? (in https://github.com/devcontainers/features/blob/21b034ed57a0b3c09ddd74e0995fe884c7566a28/test/java/install_ant_and_gradle_and_maven.sh & others? )